### DISCUSSION TITLE
弹出/抽入升级bug

### DISCUSSION THREAD

**Discussion by zgybd** [IndustrialCraftMC/IndustrialCraft2#20] [【漏洞反馈】Bugs Report]
1.弹出/抽入升级会导致游戏崩溃
2.流体抽入/弹出升级 未生效
<img width="902" alt="a0b328abd7d64025c20ee7ba2b22d05" src="https://user-images.githubusercontent.com/69385074/236509101-f39ffe79-d441-49a1-8ff5-5925b3005be4.png">
<img width="937" alt="84cb976170a7fd163b845201f0aadb5" src="https://user-images.githubusercontent.com/69385074/236509127-aef4977d-6822-4e02-ab90-d51bade59146.png">

**Comment by zgybd** [comment]
[错误报告-2023-5-5_23.54.18.zip](https://github.com/AbbottMc/IndustrialCraft2/files/11408128/-2023-5-5_23.54.18.zip)

**Comment by AbbottMc** [comment]
已修复
